### PR TITLE
fix 3968

### LINKED ISSUE #3968: Need add native isChecked, isDisabled Playwright methods
Locator class has native methods, that could be add to Playwright Helper
this is an improvement

### PR DESCRIPTION
**Pull request by hatufacci**
## Motivation/Description of the PR
- Description of this PR, which problem it solves
- Resolves #3968 

Applicable helpers:

- [X] Playwright

## Type of change

- [x] :rocket: New functionality
- [x] :clipboard: Documentation changes/updates

## Checklist:

- [X] Tests have been added
- [x] Documentation has been added (Run `npm run docs`)
- [X] Lint checking (Run `npm run lint`)
- [x] Local tests are passed (Run `npm test`)

### CODE REVIEW

File: lib/helper/Playwright.js [hunk 1]
```diff
@@ -993,6 +993,33 @@ class Playwright extends Helper {
     await el.blur(options);
     return this._waitForAction();
   }
+  /**
+   * {{> isElementChecked }}
+   * @param {object} See https://playwright.dev/docs/api/class-locator#locator-is-checked
+   * @return {Promise<boolean>}
+   *
+   */
```
Comment (kobenguyent) [inline lib/helper/Playwright.js:1003]: basically this jsdocs would be a hyperlink to `.mustache` file to render the docs when you run `npm run docs`. Please consider either creating a `.mustache` or just do something like this https://github.com/codeceptjs/CodeceptJS/blob/0d74d8e72de5f06bddc0b18a0e55f19f6c7ae29a/lib/helper/Playwright.js#L1281

File: lib/helper/Playwright.js [hunk 2]
```diff
@@ -993,6 +993,33 @@ class Playwright extends Helper {
...
+
+  async isElementChecked(locator, options = {}) {
```
Comment (kobenguyent) [inline lib/helper/Playwright.js]: well the function name alone sounds okay but when using as `I.isElementChecked` sounds a bit strange. May we consider another name? Like `grabCheckedElementStatus`?

File: lib/helper/Playwright.js [hunk 3]
```diff
@@ -993,6 +993,33 @@ class Playwright extends Helper {
...
+    throw new Error('Element is not a checkbox or radio input');
+  }
+  /**
+   * {{> isElementDisabled }}
+   * @param {object} See https://playwright.dev/docs/api/class-locator#locator-is-disabled
+   * @return {Promise<boolean>}
+   *
+   */
+
+  async isElementDisabled(locator, options = {}) {
```
Comment (kobenguyent) [inline lib/helper/Playwright.js]: same here.

File: lib/helper/Playwright.js [hunk 4]
```diff
@@ -993,6 +993,33 @@ class Playwright extends Helper {
...
+      return el.isChecked(options);
+    }
+    throw new Error('Element is not a checkbox or radio input');
+  }
+  /**
+   * {{> isElementDisabled }}
+   * @param {object} See https://playwright.dev/docs/api/class-locator#locator-is-disabled
+   * @return {Promise<boolean>}
+   *
+   */
```
Comment (kobenguyent) [inline lib/helper/Playwright.js:1022]: same above concern!

File: lib/helper/Playwright.js [hunk 5]
```diff
@@ -993,6 +993,33 @@ class Playwright extends Helper {
...
+
+  async isElementDisabled(locator, options = {}) {
+    const el = await this._locateElement(locator);
+    return el.isDisabled(options);
+  }
```
Comment (kobenguyent) [inline lib/helper/Playwright.js:1027]: what if element is not found?

Comment (hatufacci) [reply lib/helper/Playwright.js:1027]: @kobenguyent it fails with standard error like in findElement()
`async function findElement(matcher, locator) {
  if (locator.react) return findReact(matcher, locator);
  locator = new Locator(locator, 'css');

  return matcher.locator(buildLocatorString(locator)).first();
}`

File: lib/helper/Playwright.js (hunk 2 above)
Comment (hatufacci) [reply lib/helper/Playwright.js]: @kobenguyent fixed. changed to grabCheckedElementStatus, grabDisabledElementStatus

File: lib/helper/Playwright.js (hunk 3 above)
Comment (hatufacci) [reply lib/helper/Playwright.js]: @kobenguyent fixed. changed to grabCheckedElementStatus, grabDisabledElementStatus

File: lib/helper/Playwright.js (hunk 4 above)
Comment (hatufacci) [reply lib/helper/Playwright.js:1022]: @kobenguyent run npm run docs

File: lib/helper/Playwright.js (hunk 1 above)
Comment (hatufacci) [reply lib/helper/Playwright.js:1003]: @kobenguyent run npm run docs

**Review by kobenguyent — APPROVED** [review]

**Pull request merged**